hello-world/tutorial/index.md

```diff
@@ -0,0 +1,471 @@
+			packetAck.PostID,
+			data.Title,
+			packet.DestinationPort+"-"+packet.DestinationChannel,
+		)
+
+		return nil
+	default:
+		// The counter-party module doesn't implement the correct acknowledgment format
+		return errors.New("invalid acknowledgment format")
+	}
+}
+```
+
+### Storing information about timed out packet
+
+In `timedoutPost` posts we store posts that have not been received by target chains. It follows the same format as `sentPost` 
+
+```go
+// OnTimeoutIbcPostPacket responds to the case where a packet has not been transmitted because of a timeout
+func (k Keeper) OnTimeoutIbcPostPacket(ctx sdk.Context, packet channeltypes.Packet, data types.IbcPostPacketData) error {
+
+	k.AppendTimedoutPost(
+		ctx,
+		data.Creator,
+		data.Title,
+		packet.DestinationPort+"-"+packet.DestinationChannel,
+	)
+
+	return nil
+}
+```
+
+That is our basic `blog` module setup.
+The blockchain is now ready!
+
+Let's spin it up and send a blog post from one blockchain app to the other.
+We will need various terminal windows for this.
+
+## IBC module tests
+
+### Starting the chains
+
+To test the IBC module, we will serve with starport two blockchain networks on the same machine. Both blockchains uses the same source code but with a different chain ID
+
+We will have a blockchain network named `earth` and the other name `mars`
+
+We need the two following files in the project directory:
+
+`planet/earth.yml`
+
+```yaml
+accounts:
+  - name: alice
+    coins: ["1000token", "100000000stake"]
+  - name: bob
+    coins: ["500token"]
+validator:
+  name: alice
+  staked: "100000000stake"
+faucet:
+  name: bob
+  coins: ["5token"]
+genesis:
+  chain_id: "earth"
+init:
+  home: "$HOME/.earth"
+```
+
+`planet/mars.yml`
+
+```yaml
+accounts:
+  - name: alice
+    coins: ["1000token", "1000000000stake"]
+  - name: bob
+    coins: ["500token"]
+validator:
+  name: alice
+  staked: "100000000stake"
+faucet:
+  port: 4501
+  name: bob
+  coins:
+    - "5token"
+servers:
+  rpc-address: "0.0.0.0:26659"
+  p2p-address: "0.0.0.0:26658"
+  prof-address: "0.0.0.0:6061"
+  grpc-address: "0.0.0.0:9091"
+  api-address: "0.0.0.0:1318"
+  frontend-address: "0.0.0.0:8081"
+  dev-ui-address: "0.0.0.0:12346"
+genesis:
+  chain_id: "mars"
+init:
+  home: "$HOME/.mars"
+```
+
+We can then run the two blockchain networks with the following commands on two separate terminals:
+
+```
+starport serve -c earth.yml
+starport serve -c mars.yml
+```
+
+### Starting the relayer
+
+First we need to configure the relayer, we use the configure command of starport with the `--advanced` option:
+
+```
+starport relayer configure -a --source-rpc "http://0.0.0.0:26657" --source-faucet "http://0.0.0.0:4500" --source-port "blog" --source-version "blog-1" --target-rpc "http://0.0.0.0:26659" --target-faucet "http://0.0.0.0:4501" --target-port "blog" --target-version "blog-1"
+```
+
+The output will be like:
+
+```
+---------------------------------------------
+Setting up chains
```

Comment:
```suggestion
Set up chains
```

Reply:
This indeed is feedback from Starport 
https://github.com/tendermint/starport/blob/7a221bf9a423b6b66fb19c0786c26ff82b9dec38/starport/interface/cli/starport/cmd/relayer_configure.go#L62